src/openMVG/multiview/solver_resection_p4pf.cpp

```diff
@@ -0,0 +1,1231 @@
+#include "openMVG/multiview/projection.hpp"
+#include "openMVG/numeric/numeric.h"
+#include "openMVG/multiview/solver_resection_p4pf.hpp"
+
+#include <cmath>
+#include <cstring>
+#include <limits>
+#include <iostream>
+
+namespace openMVG {
+namespace resection {
+
+void GJ(double *A, int rcnt, int ccnt, double tol)
+{
+  int row = 0;
+  int col = 0;
+  int k;
+  int l;
```

Comment:
`l` and `k`, beside the naming issue, can be safely removed here and declared inside each for